docs/source/examples/unbounded.py

```diff
@@ -0,0 +1,22 @@
+from gpkit import Variable, Model, settings
+from gpkit.tools import BoundedConstraintSet
+
+A = Variable("A")
+D = Variable("D")
+F = Variable("F")
+mi = Variable("m_i")
+
+Fs = Variable("Fs", 0.9)
+mb = Variable("m_b", 0.4)
+V = Variable("V", 300)
+
+constraints = [F >= D + V**2,
+               D == V**2*A,
+               V >= mi + mb,
+               Fs <= mi,
+               ]
+
+# Model(F, constraints).solve("mosek")  # returns UNKNOWN
+if "mosek" in settings["installed_solvers"]:
+    sol = Model(F, BoundedConstraintSet(constraints)).solve("mosek")
+    print sol["boundedness"]
```

Comment:
`docs/source/examples/unbounded_output.txt` appears to be an empty file on this branch, which isn't what we want (@bqpd maybe because you're running on a machine without mosek?)